cfme/control/simulation.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from navmazing import NavigateToSibling
+from widgetastic.widget import Select
+from widgetastic_patternfly import Button
+
+from cfme.base import Server
+from cfme.base.login import BaseLoggedInPage
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep
+
+
+class ControlSimulationView(BaseLoggedInPage):
+    """Basic view for Control/Simulation tab."""
+    event_selection = Select(id="event_typ")
+    vm_selection = Select(id="filter_typ")
+    submit_button = Button("Submit")
+    reset_button = Button("Reset")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.event_selection.is_displayed and
+            self.vm_selection.is_displayed and
+            self.submit_button.is_displayed and
+            self.reset_button.is_displayed
+        )
```

Comment:
There is no results pane when you open Simulation tab, the tree with simulation results will be shown only after filling that widgets.